soql-server-pg/src/main/scala/com/socrata/pg/server/QueryServer.scala

```diff
@@ -376,8 +453,12 @@ class QueryServer(val dsInfo: DSInfo, val caseSensitivity: CaseSensitivity) exte
             NotModified(Seq(etag))
           case Some(_) | None =>
             try {
-              val (qrySchema, version, results) = runQuery(pgu, copy, analysis, rowCount, queryTimeout)
-              Success(qrySchema, copy.copyNumber, version, results, etag, lastModified)
+              runQuery(pgu, copy, analysis, rowCount, queryTimeout, explain, analyze) match {
+                case RowsQueryResult(qrySchema, version, results) =>
+                  Success (qrySchema, copy.copyNumber, version, results, etag, lastModified)
```

Comment:
I think the whitespace is unintended here?

Reply:
The newly added match adds a set of curly braces.